wiki/Beatmap_Editor/AiMod/zh.md

```diff
@@ -104,10 +116,22 @@ xxx confict...的xxx可能代表的信息：Source（来源）、Tags、Title（
 
 解决：仔细检查每个难度的 metadata，确保其一致。
 
-#### This mapset needs at least xxx diff(s)
+### This mapset needs at least xxx diff(s)
 
 翻译：**这张 beatmap 需要有至少xxx难度**
 
 ..at least xxx diff(s)的xxx可能代表的信息：1 easy/normal（需要1个 easy 或者 normal 的难度）、2 standard（需要2个难度）
 
 级别：警告
+
+
+## 缺陷
+- **AiMod 无法检测许多在[Ranking Criteria](/wiki/Ranking_Criteria)中被视为违反规则的问题。** 例如未对齐节拍的物件、击打音效延迟、谱面背景图片大小超过2.5MB等。
+- **AiMod 可能会警告您一些不是问题的问题。** 例如[马拉松图](/wiki/Beatmap/Marathon)应当有两个难度，[Kiai](/wiki/Kiai_Time)时长少于 15 秒。
+- **如果启用了 `Check distance snap` 功能，AiMod 将不会检查 Combo 间的距离。**
+- **如果beatmap文件夹里面有多个MP3文件，AiMod可能会警告您您有多个歌曲文件**，虽然这些mp3可能会用作在Hitsound 比如"掌声" <!-- 这句话的后面不太好理解 -->
+- **选择物件时不要打开AiMod,尤其是Spinner** <!-- 此处证据可以自己在Edit模式下选择Spinner的时候打开AiMod -->
```

Comment:
描述错误，该问题实际出现在选择待放置物件（打击圈、滑条、转盘）时，而非选择物件时。
Incorrect description, the issue only happened while going to place a hit object (circle, slider, and spinner). Rather than "while selecting a hit object"